src/script/hydra-send-stats

```diff
@@ -2,14 +2,15 @@
 
 use strict;
 use utf8;
+use Hydra::Helper::Nix;
 use Net::Statsd;
 use File::Slurp;
 use JSON;
 
 STDERR->autoflush(1);
 binmode STDERR, ":encoding(utf8)";
 
-my $config = getHydraConfig();
+my $config = Hydra::Helper::Nix::getHydraConfig();
```

Comment:
`use` brings it in to scope by default, so this part isn't necessary. I'll fix this up in a push I'm about to make adding some tests. Thanks!

Reply:
(I know I made this mistake one line lower. I know more perl than I did then :'))